main.js

```diff
@@ -1,13 +1,21 @@
-var outfit = new Outfit({id: generateRandomId()});
+// Outfit class instance
+var outfit = new Outfit(generateRandomId());
 
 function generateRandomId() {
   return Math.floor(Math.random() * 1000);
 }
-//button lists
+console.log(outfit)
```

Comment:
Let's go ahead and delete this! :)

Reply:
Whoops! I know I missed a couple of these!